dkg/pedersen/json/json.go

```diff
@@ -476,8 +478,7 @@ func encodeReshare(msg types.Reshare) (Message, error) {
 	return Message{Reshare: &dr}, nil
 }
 
-func (f msgFormat) decodeReshare(ctx serde.Context,
-	msg *Reshare) (serde.Message, error) {
+func (f msgFormat) decodeReshare(msg *Reshare) (serde.Message, error) {
```

Comment:
why remove the context here and replace it with an ignored parameter (`_`) everywhere else? 
It seems inconsistent 🤔